draft-ietf-quic-recovery.md

```diff
@@ -433,11 +433,11 @@ When crypto packets are outstanding, the TLP and RTO timers are not active.
 #### Retry and Version Negotiation
 
 A Retry or Version Negotiation packet causes a client to send another Initial
-packet, effectively restarting the connection process.
-
-Either packet indicates that the Initial was received but not processed.
-Neither packet can be treated as an acknowledgment for the Initial, but they MAY
-be used to improve the RTT estimate.
+packet, effectively restarting the connection process and resetting connection
+state, including resetting any pending timers.  Either packet indicates that the
+Initial was received but not processed.  Neither packet can be treated as an
+acknowledgment for the Initial, but they MAY be used to improve the RTT
+estimate.
```

Comment:
Here we're into dangerous waters again.  Resetting most of the state is fine, but you can't forget the connection ID and token give to you in Token, or the list of versions from the Version Negotiation.  Here, only RTT is mentioned.  If this were scoped to recovery and congestion state, that might be better than having to reiterate what is carried over.